tests/integration/page-router.test.ts

```diff
@@ -126,6 +126,10 @@ test<FixtureTestContext>('Should serve correct locale-aware custom 404 pages', a
     load(responseImplicitDefaultLocale.body)('[data-testid="locale"]').text(),
     'Served 404 page content should use default locale if locale is not explicitly used in pathname (after basePath)',
   ).toBe('en')
+  expect(
+    responseImplicitDefaultLocale.headers['netlify-cdn-cache-control'],
+    'Response for not existing route if locale is not explicitly used in pathname (after basePath) should have 404 status',
```

Comment:
ops this is not right msg - just copied some `expect()` line from before and forgot to adjust it (and same for other assertions here)